Vitesse/Vitesse/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+@MainActor
+final class LoginViewModel: ObservableObject {
+    @Published var email = ""
+    @Published var password = ""
+    @Published var showAlert = false
+    @Published var isLoading = false
+    @Published var alertMessage: String = ""
+    @Published var isLogged = false
```

Comment:
Tu dupliques ta Source Of Truth avec cette var. Ta seule source of truth devrait être ta var `isLogged` dans ton `VistesseApp`